hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1158,7 +1160,17 @@ def set_environment_variables_for_multi_node() -> None:
     """
     Sets the environment variables that PyTorch Lightning needs for multi-node training.
     """
-    if ENV_AZ_BATCHAI_MPI_MASTER_NODE in os.environ:
+    if ENV_AZ_BATCH_MASTER_NODE in os.environ:
```

Comment:
for easier diagnostics, should we print all environment variables via logging.debug?